tests/foreman/longrun/test_inc_updates.py

```diff
@@ -212,3 +214,101 @@ def test_positive_noapply_api(
+            reposet=REPOSET[_repo],
+            releasever=REPOS[_repo]['releasever'],
+        )
+        module_target_sat.cli.ContentView.add_repository(
+            {
+                'id': cv['id'],
+                'organization-id': module_sca_manifest_org.id,
+                'repository-id': rh_repo_id,
+            }
+        )
+        module_target_sat.api.Repository(id=rh_repo_id).sync(synchronous=False)
+
+    # wait for all repo sync tasks
+    sync_tasks = module_target_sat.wait_for_tasks(
+        search_query=(
+            'label = Actions::Katello::Repository::Sync'
+            f' and started_at >= "{repo_sync_timestamp}"'
+        ),
+        search_rate=10,
+        max_tries=200,
+    )
+    assert all(task.poll()['result'] == 'success' for task in sync_tasks)
+    # publish and fetch new CVV
+    module_target_sat.cli.ContentView.publish({'id': cv['id']})
+    content_view = module_target_sat.cli.ContentView.info({'id': cv['id']})
+    cvv = content_view['versions'][0]
+
+    # update incremental version via hammer, using one errata.
+    # expect: incr. "version-1.1" is created
+    update_start_time = datetime.utcnow()
+    result = module_target_sat.cli.ContentView.version_incremental_update(
+        {'content-view-version-id': cvv['id'], 'errata-ids': REAL_RHEL8_1_ERRATA_ID}
+    )
+    assert 'version-1.1' in str(result[0].keys())
+    update_duration = (datetime.utcnow() - update_start_time).total_seconds()
+    logger.info(
+        f'Update of incremental version-1.1, for CV id: {content_view["id"]},'
+        f' took {update_duration} seconds.'
+    )
+    # publish the full CV, containing the added version-1.1
+    publish_start_time = datetime.utcnow()
+    result = module_target_sat.cli.ContentView.publish({'id': cv['id']})
+    publish_duration = (datetime.utcnow() - publish_start_time).total_seconds()
+    logger.info(f'Publish for CV id: {content_view["id"]}, took {publish_duration} seconds.')
+    # some arbritrary timeouts, given amount of content in CV from repos.
+    assert update_duration <= 60
+    assert publish_duration <= 240
+    # Per BZs: expect update duration was quicker than publish duration,
+    # if instead, update took longer, check that they were close,
+    # that update did not take ~significantly more time.
+    if update_duration >= publish_duration:
+        # unexpected: perhaps both tasks were very quick, took a handful of seconds,
+        # assert the difference was not significant (within 20%).
+        assert (update_duration - publish_duration) / publish_duration <= 0.2, (
+            f'Incremental update took longer than publish of entire content-view id: {content_view["id"]}:'
+            f' Update took significantly more time, 20% or longer, than publish.'
+            f' update duration: {update_duration} s.\n publish duration: {publish_duration} s.'
+        )
+    # else: base expected condition: update duration was quicker than publish.
```

Comment:
Do we need to add anything here?

Reply:
For this, I just put a docstring that the expected condition was found via the prior if statement.

I would say if we want to, we could add some extra logging of update vs publish and deltas. 
I wouldn't say assertions are needed from this point on, since the prior `if` checked the condition for us.

Reply:
@Griffin-Sullivan - I added the arbitrary timeout assertions to the bottom, as they would point more to some sort of performance degradation. 
After some discussion with @vsedmik !

Reply:
Nice comment helping others to understand testing merit.